### DISCUSSION TITLE
Using Filament Form Builder outside admin area

### DISCUSSION THREAD

**Discussion by Romaixn** [filamentphp/filament#8729] [Help] [answered]
Hey everyone !
I'd like to use Form Builder to create a form on the front side (not in Filament admin). I've done it, but I'm missing some Alpine JS (especially for the FileUpload).
Do you know if I'm missing an import? Or if it's just not possible?

Because at the moment, the FileUpload code looks like this, but the JS code doesn't seem to be called by itself...
![image](https://github.com/filamentphp/filament/assets/18333296/4ed05e65-b159-4cdc-8697-ea61822c1600)

**Comment by leandrocfe** [comment, marked as answer]
Did you do [this setup](https://filamentphp.com/docs/3.x/forms/installation)?
This js should load this file `public/js/filament/forms/components/file-upload.js`
